sdk/src/main/java/com/ericsson/research/owr/sdk/RtcCandidates.java

```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright (c) 2015, Ericsson AB.
+ * Copyright (c) 2015, Paul-Louis Ageneau
```

Comment:
I originally wrote this in december and restarted tampering with it only recently, so 2016 is OK.